pages/index.js

```diff
@@ -278,6 +278,12 @@ const FlexWrap = styled.div`
     padding: 3rem;
     text-align: center;
   }
+
+  @media (max-width: ${props => props.theme.breakpoints.smallMax}) {
+    .mission-statement {
+      margin-top: 6rem;
+    }
+  }
```

Comment:
This will work fine for the homepage, but we will still have the issue on other pages.

What I think we should do here is add a dummy element after the `<header>` that is 90px tall on mobile and 0px tall once we are in desktop mode.

[That would be added here](https://github.com/texas-justice-initiative/website-nextjs/blob/58c8835a8467ba6f00f0ac8577fd9dbecc927f44/components/Page.js#L35) and should look something like:

```
<div style="height:90px;width:100%;"></div>
```

Note that this should probably be a styled component so we can reset the height to 0px for desktop.

I hope that helps. Let me know if you you are stuck and I can jump in and make a PR.